src/cronjobs

```diff
@@ -116,3 +116,47 @@ p({
     }
   """
 })
+
+p({
+  "class":"foam.nanos.cron.Cron",
+  "id":"CleanupTempFileCron",
+  "description":"Cron to clean up files which exist more than 4 hours",
+  schedule:{
+    class:"foam.nanos.cron.IntervalSchedule",
+    duration:{
+      class:"foam.nanos.cron.TimeHMS",
+      hour:4,
+      minute:0,
+      seconds:0
+    }
+  },
+  "code": """
+    import foam.core.X;
+    import foam.dao.ArraySink;
+    import foam.dao.DAO;
+    import foam.nanos.fs.TempFile;
+    import java.io.File;
+    import java.util.Calendar;
+    import java.util.Date;
+    import java.util.List;
+
+    DAO tempFileDAO = (DAO) x.get("tempFileDAO");
+
+    Calendar calendar = Calendar.getInstance();
+    calendar.setTime(new Date());
+    calendar.add(Calendar.HOUR_OF_DAY, -4);
+    Date forHourBefore = calendar.getTime();
+
+    List tempFiles = ((ArraySink) tempFileDAO.select(new ArraySink())).getArray();
```

Comment:
Additionally this could simply be 
where(LT(CREATED, threshold).select().removeAll();

Reply:
I can't do that if I want to remove file from file system. 
Do I need to? @lchanmann

Reply:
You can add TempFileDAO decorator to handle removing the file from file system then can just call removeAll() in the cron job.

Reply:
I'm fine with walking the list returned by select, but the where/select can handle the conditional testing.